types/index.d.ts

```diff
@@ -5920,6 +5944,28 @@ declare namespace math {
       a2: MathCollection
     ): MathJsChain<T>
 
+    /*************************************************************************
+     * Signal functions
+     ************************************************************************/
+    /**
+     * Compute the transfer function of a zero-pole-gain model.
+     */
+    zpk2tf<T extends MathCollection>(
+      this: MathJsChain<T>, // chained variable will be used as z
+      p: T,
+      k?: number
+    ): MathJsChain<T>
+
+    /**
+     * Calculates the frequency response of a filter given its numerator and denominator coefficients.
+     */
+    freqz<T extends number | MathArray | MathArray[]>(
+      this: MathJsChain<T>,
+      b: T,
+      a: T,
+      w?: T | number
+    ): MathJsChain<object>
```

Comment:
I think the output is not a `MathJsChain<object>` but `MathJsChain<{ w: T; h: T }>`, right?

Reply:
Yes